### ISSUE TITLE
Adjust the cookie policy

### ISSUE THREAD

**Issue by mukrop** [faktaoklimatu/web-core#128]
This applies to all our websites (as all use analytics): faktaoklimatu.cz, faktyoklime.sk, factsonclimate.org, 2050podcast.cz

From 1. 1. 2022, the Czech legislation is changing to harmonize with the rest of the EU and GDPR. Explicit opt-in will be strictly necessary (as opposed to out-out) with clear user information, equally accessible accept/reject options and the possibility to adjust the consent later.

Check existing solutions (e.g. [Cookiebot](https://manage.cookiebot.com)), we may not need to reinvent the wheel.

The expected solution will feature the following:
* Visually unobtrusive cookie banner on our pages.
  * Lower left/right, bubble-like, maybe share the design with the development notice?
  * Disable all cookies in the development preview and this information to the development information?
* In the cookie bar: short information, options of accept all/reject all.
  * Preferences may not be necessary, we probably only use necessary cookies (it at all) and analytical cookies.
* An option to change one's consent.
  * Probably include in the GDPR page.
* Legally, we should store the consents given to be able to proove they were given (do not reinvent the wheel!).
* Nice to have: More detailed information on cookie types, processing goals and retention periods.
  * Probably include in the GDPR page.
* Technicalities
  * The analytics should load only after consent is given. DNT option should still be honored. (Check Google Tag Manager options.)
  * Check what information is sent to EcoMail and if they store any cookies on our site.
  * Try to collect how many people accepted/rejected cookies, if possible.
* Important: How not to display the cookie bar for 1) people who had consented in the past and 2) (more importantly) people who had rejected in the past?
